index.html

```diff
@@ -173,8 +173,35 @@ <h3 class="about-languages">Languages</h3>
                         <img class="line" src="images/Separator-Bottom.svg" alt="">
                     </ul>
                 </div>
+                <footer class="contact-form">
+                    <section class="contact-form-details">
+
+                        <h2 class="contact-title">Contact me</h2>
+                        <p class="contact-details">If you have an application you are interested in developing, 
+                        a feature that you need built or a project that needs coding.     
+                        I’d love to help with it </p>
+
+                        <form action="https://formspree.io/f/xrgdkowz" method="post">
+                            <ul class="contact-form-list">
+                                <li class="contact-name"><input type="text" name="name" placeholder="Full Name" maxlength="30" required></li>
+                                <li><input type="email" name="email" placeholder="Email" required></li>
+                                <li><textarea type="text" name="message" id="" cols="30" rows="10" placeholder="Please write your message here"
+                                    maxlength="500" required></textarea></li>
+                                <li><button type="submit" class="contact-form-button">Get in Touch</button></li>
+                            </ul>
+                            <img class="contact-form-end-image" src="images/Shape.svg" alt="">
```

Comment:
- [OPTIONAL] You can remove this line because this was only showing that the design is for mobile. We do not necessarily show it on the browser.

Reply:
Hello @levy002 

Thank you for the review.

Do you mean I should remove the line at the bottom of the contact form?

am I right?

Reply:
Yes, you are right @Hamdullah-Andar

Reply:
Ok got it 

I'll remove the line, merge it and submit it

Thank you so much